src/components/Activities/Activity1/PersonCompConvoAnimation.tsx

```diff
@@ -0,0 +1,75 @@
+import React, { useContext } from 'react';
+
+import AppleSvg from '../../../assets/activity1/apple.svg';
+import CharacterSvg from '../../../assets/activity1/character3.svg';
+import ComputerSvg from '../../../assets/activity1/computer.svg';
+import LemonSvg from '../../../assets/activity1/lemon.svg';
+import PartyConfettiSvg from '../../../assets/activity1/party_confetti.svg';
+
+import { CarouselContext } from '../../shared/Carousel';
+import { TextBubbleStyles } from '../../shared/PlaynetConstants';
+import Convo, { Phrase } from './Convo';
+
+
+interface PersonCompConvoAnimationProps {
+  timeBtwnWords: number;
+}
+
+const PersonCompConvoAnimation = (props: PersonCompConvoAnimationProps): JSX.Element => {
```

Comment:
nit: name is a tad long
```
```suggestion
const ComputerConversation= (props: ComputerConversationProps): JSX.Element => {
```

Reply:
ok